### ISSUE TITLE
Error when creating Server Profile Template with multiple logical disks

### ISSUE THREAD

**Issue by DungKHoang** [HewlettPackard/POSH-HPOneView#196]
Please fill in as much information as possible to help resolve your issue.

### Expected Behavior
Create server profile template with 1 disk controller and 2 logical disks

### Actual Behavior

When creating a server profile template with local storage, when the disk controller has more than 2 disks, the new-hpovseverprofiletemplate generates error.

### Steps to reproduce
```PowerShell
$Disk1 =  New-HPOVServerProfileLogicalDisk -name 'MyDisk1'
$Disk2 = New-HPOVServerProfileLogicalDisk -name 'MyDisk2'
$Controller = New-HPOVServerProfileLogicalDiskController -Initialize -LogicalDisk $Disk1,$Disk2
New-HPOVServerProfileTemplate -Name Test -ServerHardwareType $THisSHT -LocalStorage  -EnclosureGroup $thiseg -StorageController $controller
```
### Version Information
HPE OneView PowerShell Library Version (`Get-HPOVVersion`): 3.0.1170.1773
HPE OneView Appliance Version:  3.0
[1612290925_HPOV_transcript.txt](https://github.com/HewlettPackard/POSH-HPOneView/files/677582/1612290925_HPOV_transcript.txt)

Output from `$PSVersionTable` on your Windows Host:
PSVersion                      5.1.14393.0                                                                                                                                                                               
PSEdition                      Desktop                                                                                                                                                                                   
PSCompatibleVersions           {1.0, 2.0, 3.0, 4.0...}                                                                                                                                                                   
BuildVersion                   10.0.14393.0                                                                                                                                                                              
CLRVersion                     4.0.30319.42000                                                                                                                                                                           
WSManStackVersion              3.0                                                                                                                                                                                       
PSRemotingProtocolVersion      2.3                                                                                                                                                                                       
SerializationVersion           1.1.0.1

**Comment by ChrisLynchHPE** [comment]
This is unfortunately a known issue that still has yet to be resolved.  There is no current workaround or fix yet.

**Comment by ChrisLynchHPE** [comment]
This is fixed in [Release 3.0.1191.2858](https://github.com/HewlettPackard/POSH-HPOneView/releases/tag/v3.0.1191.2858).

Closing.
